src/notifier.py

```diff
@@ -74,6 +75,8 @@ def __call__(self, configs, args):
                 sys.stdout.flush()
         except KeyboardInterrupt as e:
             self._logger.log_message(logging.INFO, "Stopping.")
+        except Exception as e:
```

Comment:
The exception object is not being used. Hence, can be dropped.
```suggestion
        except Exception:
```